projects/plugins/jetpack/modules/search/class-jetpack-instant-search.php

```diff
@@ -130,35 +130,6 @@ public function load_assets_with_parameters( $path_prefix, $plugin_base_path ) {
 		Assets::enqueue_script( 'jetpack-instant-search' );
 		$this->load_and_initialize_tracks();
 		$this->inject_javascript_options();
-
-		// It only inlines the translations for the script, but does not actually load the script.
-		$this->inject_translation_for_script(
-			plugins_url(
-				$path_prefix . '_inc/build/instant-search/jp-search.chunk-main-payload.js',
-				$plugin_base_path
-			)
-		);
-	}
-
-	/**
-	 * Add inline translations for script `$payload_url` before loading `$before_handle` script.
-	 *
-	 * @param string $payload_url - The payload url for which we load the translations.
-	 * @param string $before_handle - Inline the translations before this handle.
-	 */
-	protected function inject_translation_for_script( $payload_url, $before_handle = 'jetpack-instant-search' ) {
-		// Set a random name for the script.
-		$handle = 'jetpack-instant-search-' . wp_unique_id();
-		// Then register it, which is required for the next steps.
-		// phpcs:ignore WordPress.WP.EnqueuedResourceParameters.NoExplicitVersion
-		wp_register_script( $handle, $payload_url, array(), false, false );
-		// Set translation domain to `jetpack`, and we need to explicitly set the `path` to load translations files for WPCOM.
-		// Otherwise WPCOM would try to load from `WP_LANG_DIR . '/mu-plugins'` and fails.
-		wp_set_script_translations( $handle, 'jetpack', WP_LANG_DIR . '/plugins' );
```

Comment:
in WPCOM, the translation path was resolved to `mu-plugins` if `/plugins` not specified here.

Reply:
It should be ok since the path now isn't being determined by wpcom PHP code, but testing is a good idea.

Looks like we may have to wait until #21932 is merged first so the TC build on D70962-code will succeed. Or merge this into #21932 then test it with a D70957-code that contains the changes from here too, if reviewers would rather.

Reply:
So it turns out testing this is kind of a pain, because the Fusion patch doesn't include the changes to the Assets package but those are required for it to work.

Once I manually apply those in my sandbox, it *almost* works. It tries to fetch the right path, but apparently wpcom blocks access to `.json` files inside `wp-content/languages/`.

Reply:
Figured it out, and updated the testing instructions so it should be easy for someone else to test it too.